debian/control

```diff
@@ -0,0 +1,22 @@
+Source: libpam-policycache
+Priority: extra
+Maintainer: Nikki VonHollen <vonhollen@google.com>
+Build-Depends: debhelper (>= 8.0.0), autotools-dev, libglib2.0-dev,
+ libpam0g-dev, automake, libtool, autogen, libscrypt-dev
+Standards-Version: 3.9.2
+Section: libs
+Homepage: https://github.com/google/libpam-policycache
+
+Package: libpam-policycache
+Section: libs
```

Comment:
Section: admin

Also it should be enough to say that in the top-most stanza and then omit it here.